CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 5.0.0 - *2021-05-27*
```

Comment:
```suggestion
## 5.0.0 - *2021-05-27*
```

Our releaser will automatically do this 👍🏼

Reply:
Fixed